api/app/Checkers/ProtectedRequestUserChecker.php

```diff
@@ -56,6 +57,8 @@ public function currentUserHasRole(
         bool $requireAll = false
     ): bool {
         if (! $this->isSafeToUseRole($name)) {
+            Log::warning('Tried to unsafely use role '.json_encode($name));
```

Comment:
This is not directly related to the AC - just that it would have saved me lots of time if it existed before.  :face_exhaling: